src/index.tsx

```diff
@@ -15,21 +15,26 @@ const DEFAULT_VISIBILITY_RATIO_THRESHOLD = 0.5;
 const DEFAULT_VISIBILITY_TIME_THRESHOLD = 1000;
 
 export interface Impression {
-  insertionId: string;
+  insertionId?: string;
+  contentId?: string;
   impressionId: string;
 }
 
 interface TrackerArguments {
   /* A quick way to disable the hook. */
   enable?: boolean;
```

Comment:
Just wondering why this is optional? Could it default to true instead?

Reply:
Yea, it defaults to true.  In most cases, I expect people to just leave it on.  I updated the comments.